web_chatter_right/static/src/css/chatter_right.css

```diff
@@ -0,0 +1,23 @@
+@media only screen and (min-width: 1024px) {
+  .o_form_sheet_bg.o_form_sheet_chatter_right {
+    width: calc(100% - 390px);
+    display: inline-block;
+  }
+
+  .oe_chatter.o_chatter_right {
+    display: inline-block !important;
+    width: 390px !important;
+    min-width: 390px !important;
```

Comment:
Do not use `!important`. Those rules are extremely hard to theme. Better, use more specific selectors if you need to override some odoo css.

Also, use Less if possible.